### PR TITLE
Use node image as base image.

### PR DESCRIPTION
**Pull request by auterium**
By using the node docker image as the base image, installing node is no longer required as part of the build process

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -55,7 +50,6 @@ ONBUILD COPY . $APP_SOURCE_DIR
 # install all dependencies, build app, clean up
 ONBUILD RUN cd $APP_SOURCE_DIR && \
   $BUILD_SCRIPTS_DIR/install-deps.sh && \
-  $BUILD_SCRIPTS_DIR/install-node.sh && \
```
Comment (jfresco) [inline Dockerfile:58]: If the file `scripts/install-node.sh` is no longer necessary, you could delete it.

**Review by jfresco — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jfresco) [reply Dockerfile:58]: And you may want to remove the references to custom Node version installations in the `README.md` file.

Comment (auterium) [reply Dockerfile]: Done

Comment (auterium) [reply Dockerfile:58]: Done

**Review by jfresco — APPROVED** [review]
🚀

**Pull request merged**